lumigator/python/mzai/jobs/evaluator_lite/evaluator_lite/tests/conftest.py

```diff
@@ -0,0 +1,23 @@
+import json
+from pathlib import Path
+
+import pytest
+
+
+def load_json(path: Path) -> dict:
+    with Path.open(path) as file:
+        return json.load(file)
+
+
+def resources_dir() -> Path:
+    return Path(__file__).parent / "data"
+
+
+@pytest.fixture(scope="session")
+def json_config_minimal() -> dict:
```

Comment:
Are these used in the tests? (Maybe the corresponding test_* was not committed?)